server/fleet/datastore.go

```diff
@@ -663,9 +663,7 @@ type Datastore interface {
 	// ReplaceVPPInstallVerificationUUID replaces the verification command UUID for all
 	// VPP app install attempts were related to oldVerifyUUID.
 	ReplaceVPPInstallVerificationUUID(ctx context.Context, oldVerifyUUID, verifyCommandUUID string) error
-	// GetAcknowledgedMDMCommandsByHost gets all commands of the given type that are in the
-	// "Acknowledged" state.
-	GetAcknowledgedMDMCommandsByHost(ctx context.Context, hostUUID, commandType string) ([]string, error)
+	GetPendingVerifyVPPInstallCommandsByHost(ctx context.Context, hostUUID string) (bool, error)
```

Comment:
```suggestion
	IsHostPendingVPPInstallVerification(ctx context.Context, hostUUID string) (bool, error)
```

Given that we don't actually get any commands here